README.adoc

```diff
@@ -812,7 +812,7 @@ Use `not=` instead of `(not (= ...))`.
 
 === `printf` [[printf]]
 
-Use `printf` instead of `(print (format ...))`.
+Use `printf` instead of `(println (format ...))`.
```

Comment:
Does `printf` insert a newline?

Reply:
No, I chose it for consistency with the example following.

Reply:
Ah, I see what you mean now. The rule was meant to imply a generic combination of `print(ln` and `format`, so I'd go with some different wording instead.

Reply:
I see, in that case it might be better to show both.